client/memez/projects/memez/src/app/components/like-dialog-box/like-dialog-box.component.html

```diff
@@ -1,6 +1,11 @@
 <ul class="liked-user-list"
-    *ngFor="let user of data">
-  <li class="liked-user">{{user.name}}</li>
+
+>
+  <li *ngFor="let like of data"
+      class="liked-user">
+    <div class="like-user-name">{{like.userLiked.name}}</div>
```

Comment:
userLiked is a bit confusing. 
I think like.userId is cleaner but that's just personal preference... whatever you prefer